reservation_units/tasks.py

```diff
@@ -26,6 +29,80 @@ def _update_reservation_unit_pricings() -> None:
     logger.info(f"Updated {num_updated} reservation units with date {today}")
 
 
+@app.task(name="update_reservation_unit_pricings_tax_percentage")
+def update_reservation_unit_pricings_tax_percentage(
+    change_date: str,
+    current_tax: str,
+    future_tax: str,
+    ignored_company_codes: list[str] | tuple[str] = (),
+) -> None:
+    from reservation_units.models import ReservationUnitPricing, TaxPercentage
+
+    SentryLogger.log_message(
+        message="Task `update_reservation_unit_pricings_tax_percentage` started",
+        details=f"Task was run with "
+        f"change_date: {change_date}, "
+        f"current_tax: {current_tax}, "
+        f"future_tax: {future_tax}, "
+        f"ignored_company_codes: {ignored_company_codes}",
```

Comment:
I find multiline strings as keyword arguments a bit hard to read without parenthesis:

```
details=(
    f"Task was run with "
    f"change_date: {change_date}, "
    f"current_tax: {current_tax}, "
    f"future_tax: {future_tax}, "
    f"ignored_company_codes: {ignored_company_codes}",
 )
```

Makes it hard to see when the next keyword argument begins. But, it's just a preference.